packages/lesswrong/server/vulcan-lib/apollo-server/initGraphQL.ts

```diff
@@ -226,6 +226,19 @@ const getFields = <N extends CollectionNameString>(schema: SchemaType<N>, typeNa
         const resolver = {
           [typeName]: {
             [resolverName]: (document: ObjectsByCollectionName[N], args: any, context: ResolverContext, info: any) => {
+              // First, check if the value was already fetched by a SQL resolver.
```

Comment:
This bit is super-important. This is where we bypass the normal "code-resolver" for any fields that have a SQL resolver.